calico_containers/calico_ctl/bgp.py

```diff
@@ -37,6 +38,39 @@
 from utils import get_container_ipv_from_arguments
 
 
+def validate_arguments(arguments):
+    """
+    Validate argument values:
+        <PEER_IP>
+        <AS_NUM>
+
+    Arguments not validated:
+
+    :param arguments: Docopt processed arguments
+    """
+    # Validate IPs
+    peer_ip_ok = arguments.get("<PEER_IP>") is None or \
+                    netaddr.valid_ipv4(arguments["<PEER_IP>"]) or \
+                    netaddr.valid_ipv6(arguments["<PEER_IP>"])
+    asnum_ok = True
+    if arguments.get("<AS_NUM>") or arguments.get("--as"):
```

Comment:
I think the --as switch is only valid on the node command - so this could be simplified a bit.